Sources/JWTKit/Utilities/OpenSSLSigner.swift

```diff
@@ -31,7 +31,7 @@ private enum OpenSSLError: Error {
 }
 
 extension CryptoSigner {
-    func digest<Plaintext>(_ plaintext: Plaintext) throws -> [UInt8]
+    public func digest<Plaintext>(_ plaintext: Plaintext) throws -> [UInt8]
```

Comment:
Please don't make any things public that don't _need_ to be, we have way too many public things already.